source/platforms/databricks.md

```diff
@@ -63,9 +65,92 @@ gdf
+
+### Create init-script
+
+To get started, you must first configure an [initialization script](https://docs.databricks.com/en/init-scripts/index.html) to install `dask`, `dask-databricks` RAPIDS libraries and all other dependencies for your project.
+
+Databricks recommends using [cluster-scoped](https://docs.databricks.com/en/init-scripts/cluster-scoped.html) init scripts stored in the workspace files.
+
+Navigate to the top-left **Workspace** tab and click on your **Home** directory then select **Add** > **File** from the menu. Create an `init.sh` script with contents:
+
+```bash
+#!/bin/bash
+set -e
+
+# The Databricks Python directory isn't on the path in
+# databricksruntime/gpu-tensorflow:cuda11.8 for some reason
+export PATH="/databricks/python/bin:$PATH"
+
+# Install RAPIDS (cudf & dask-cudf) and dask-databricks
+/databricks/python/bin/pip install --extra-index-url=https://pypi.nvidia.com \
+      cudf-cu11 \
+      dask[complete] \
+      dask-cudf-cu11  \
+      dask-cuda=={rapids_version} \
+      dask-databricks
+
+# Start the Dask cluster with CUDA workers
+dask databricks run --cuda
+
+```
+
+**Note**: By default, the `dask databricks run` command will launch a dask scheduler in the driver node and standard workers on remaining nodes.
+
+To launch a dask cluster with GPU workers, you must parse in `--cuda` flag option.
+
+### Launch Dask cluster
+
+Once your script is ready, follow the same [instructions](launch-databricks-cluster) to launch a **Multi-node** Databricks cluster.
+
+After docker setup in **Advanced Options**, switch to the **Init Scripts** tab and add the file path to the init-script in your Workspace directory starting with `/Users/<user-name>/<script-name>.sh`.
 
+You can also configure cluster log delivery in the **Logging** tab, which will write the init script logs to DBFS in a subdirectory called `dbfs:/cluster-logs/<cluster-id>/init_scripts/`. Refer to [docs](https://docs.databricks.com/en/init-scripts/logs.html) for more information.
+
+![Screenshot of init script](../images/databricks-dask-init-script.png)
+
+Now you should be able to select a "GPU-Accelerated" instance for both **Worker** and **Driver** nodes.
+
+![Screenshot of driver worker node](../images/databricks-worker-driver-node.png)
+
+### Connect to Client
+
+To test RAPIDS, Connect to the dask client and submit tasks.
+
+```python
+import dask_databricks
+
+
+client = dask_databricks.get_client()
+client
+```
+
+Dask host interactive diagnostic web servers using [Bokeh](https://docs.bokeh.org/en/latest/). The **[Dask dashboard](https://docs.dask.org/en/latest/dashboard.html)**, in particular provides a web-based UI with visualizations and real-time information about the Dask cluster's status i.e task progress, resource utilization, etc.
+
+The Dask dashboard server will start up automatically when the scheduler is created, and is hosted on ports `8087, 8787` by default.
```

Comment:
I think we should only mention `8087` here as this is the port that we are proxying in through Databricks. The default is `8787` but Databricks often runs other software including R Shiny which also defaults to `8787` so we changed to a non-standard port to avoid this.